src/main/resources/templates/db_scripts/schema.json

```diff
@@ -698,7 +698,7 @@
     {
       "run": "after",
       "snippetPath": "populateEffectiveCallNumberComponentsForExistingItems.sql",
-      "fromModuleVersion": "17.1.0"
+      "fromModuleVersion": "18.2.0"
```

Comment:
Hmm. How does this work, does this mean that this definition will only work for version 18.2.0?

Reply:
Yes, for version >= `18.2.0`  I think.

Reply:
Yes, the populate script will be executed when upgrading from an old version that is `< 18.2.0`. For example when upgrading from 17.0.0 to 19.0.0. But not when upgrading from 18.2.0 to 19.0.0.